state/volume.go

```diff
@@ -1546,6 +1548,10 @@ func (sb *storageBackend) AllVolumes() ([]Volume, error) {
 	return volumesToInterfaces(volumes), nil
 }
 
+// volumeGlobalKey returns the global database key for the named volume.
 func volumeGlobalKey(name string) string {
-	return "v#" + name
+	return volumeKindPrefix + name
 }
+
+// volumeKindPrefix is the string we use to denote a volume kind.
+const volumeKindPrefix = "v#"
```

Comment:
These should not be exposed as the "kind" values in the log entry labels. They are a internal juju terminology that are not meant to be human readable. "u#charm" means nothing to anyone outside the juju code.

We need to define human readable labels, and can start by basing them off the names.Tag kinds, eg

"machine"
"volume"
"filesystem"
"application"
etc

We can then invent the new values as needed, eg

"machine-cloud-instance"
"unit-agent"
"unit-workload"
"unit-workload-version"

The "modification status" is used to reflect what happens when an LXD profile is applied. It can have values "idle" (the default), "error", or "applied". I am not sure we want to record "idle" here - we are really just interested when a profile has been applied, and if it did so without error. I would use the kind "machine-lxd-profile" and leave out the "idle" from the logs.